nuxt.config.js

```diff
@@ -233,7 +233,7 @@ export default {
   },
   i18n: {
     baseUrl: process.env.SITE_URL,
-    strategy: 'no_prefix',
+    strategy: 'prefix_and_default',
```

Comment:
Tks for the Pr @tupiznak , we use no_prefix because we customize the i18n. We fetch the routes from ODOO, and we don't want i18n create aditional routes for all languages, only for the NODE_LOCALE env.